### ISSUE TITLE
Rfine (item rarity) translation

### ISSUE THREAD

**Issue by DimasjkTV** [suriyun-production/mmorpg-kit-docs#105]
Old bug
Do not work refine item translation

-Eng
![image](https://user-images.githubusercontent.com/47291794/66939752-57a32080-f04c-11e9-963e-ccb201796489.png)

-Ru
![image](https://user-images.githubusercontent.com/47291794/66939800-6d184a80-f04c-11e9-9841-4947e3b1a7d4.png)

![image](https://user-images.githubusercontent.com/47291794/66939905-a2bd3380-f04c-11e9-97ef-be08f9fcda01.png)

![image](https://user-images.githubusercontent.com/47291794/66939995-cb452d80-f04c-11e9-9758-e169c63fbb41.png)

**Comment by insthync** [comment]
Okay, I will fix it

**Comment by insthync** [comment]
Fixed, you can fix it by change codes in `Item.cs` line 142 to 
```
                return "<color=#" + ColorUtility.ToHtmlStringRGB(itemRefine.titleColor) + ">" + itemRefine.Title + "</color>";
```
(Change itemRefine.title to itemRefine.Title)

**Comment by insthync** [comment]
Fixed in 1.50
